src/vmm_unix.ml

```diff
@@ -201,8 +201,7 @@ let solo5_image_target image =
 
 let solo5_tender = function Spt -> "solo5-spt" | Hvt -> "solo5-hvt"
 
-let solo5_image_devices image =
-  check_solo5_cmd "solo5-elftool" >>= fun cmd ->
+let solo5_image_devices cmd image =
```

Comment:
I don't really like moving `check_solo5_cmd` out of `solo5_image_devices` like so, but I didn't like running `check_solo5_cmd` twice either.